backend/test/users.test.js

```diff
@@ -1,219 +1,213 @@
-import models from '../database/models';
```

Comment:
Can we rename the tests folder to __tests__ and avoid having the.test file extension to match frontend and have the same convention?

Reply:
I was gonna make a separate ticket for that since this PR is already so fat

Reply:
Done